packages/odyssey-design-tokens/src/space.json

```diff
@@ -1,18 +1,15 @@
 {
   "space": {
-    "rem": {
-      "xs": { "value": "0.375rem" },
-      "s": { "value": "0.75rem" },
-      "m": { "value": "1.5rem" },
-      "l": { "value": "3rem" },
-      "xl": { "value": "6rem" }
-    },
-    "em": {
-      "xs": { "value": "0.375em" },
-      "s": { "value": "0.75em" },
-      "m": { "value": "1.5em" },
-      "l": { "value": "3em" },
-      "xl": { "value": "6em" }
+    "scale": {
+      "1": { "value": "0.28571429rem" },
```

Comment:
font scale above starts with 0 this one starts with 1 - we might give all the programmers an aneurysm here 😅

Reply:
@josesolano-okta you cool swapping this to a `0` index?

I like it for the font scale because we get `1`: `1rem` and `0` represents our smallest available size rather than having `-1` or `neg1`.

Reply:
Spoke to JS; swapping this to a 0-index, which will match our colors and font-size palettes.